examples/sshguard.conf.sample

```diff
@@ -9,12 +9,17 @@
 #BACKEND="/usr/local/libexec/sshg-fw-iptables"
 
 # Space-separated list of log files to monitor. (optional, no default)
-#FILES="/var/log/auth.log /var/log/authlog /var/log/maillog"
+# debian-style systems
+#FILES="/var/log/mail.log /var/log/auth.log"
+# slackware and redhat-style systems
+#FILES="/var/log/maillog /var/log/secure"
 
 # Shell command that provides logs on standard output. (optional, no default)
-# Example 1: ssh and sendmail from systemd journal:
+# ssh and sendmail from systemd journal
 #LOGREADER="LANG=C /usr/bin/journalctl -afb -p info -n1 -t sshd -t sendmail -o cat"
-# Example 2: ssh from os_log (macOS 10.12+)
+# mail, auth and authpriv facilities from systemd journal (https://en.wikipedia.org/wiki/Syslog#Facility)
+#LOGREADER="LANG=C journalctl -afb -p info -n1 -o cat SYSLOG_FACILITY=2 SYSLOG_FACILITY=4 SYSLOG_FACILITY=10"
```

Comment:
Have you tested this, and this works? Do the environment variables need to be given in front?